### ISSUE TITLE
Is it possible to bookmark a search?

### ISSUE THREAD

**Issue by saiwing-yeung** [brandon1024/find#348]
Most of the time when I use this extension I use it to search for one specific regex. Is it possible to bookmark it (bookmarklet?) so that I don't have to type it in every time? Thank you!

**Comment by brandon1024** [comment]
Yes! You can use the "Saved Expressions" feature. Just click the little bookmark icon in the extension popup, type an expression and click save. You can then select the extension from your saved expressions whenever you need it.
